src/main/java/com/laserfiche/api/client/model/ProblemDetails.java

```diff
@@ -0,0 +1,78 @@
+package com.laserfiche.api.client.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class ProblemDetails extends HashMap<String, Object> {
```

Comment:
Again, reminder:
1. Ensure the one in repository client is the same as this one (it's not, it has builder API and this one doesn't).
2. Repository client can probably use this class instead.

Reply:
can you specify which method in the repo client is needed, since this class in the repo client is auto-generated (i.e. they contain the schema annotation, generated annotation, and all those extra methods outside of getters and setters, which i believe you did say that i can remove in the prior PR if im not mistaken)

Reply:
@lf-jason-jia which specific method are you referring to? since the repo client version is auto-generated so i just removed both the generated and schema annotations, along with all the non getter/setter methods

Reply:
Hi @sxu1998 , that's fine. I guess we could decide on that later. I added it as a reminder for future. But I guess I should create a story for those reminders.